spectroscopy/code_src/mast_functions.py

```diff
@@ -13,7 +13,7 @@
 from astropy.table import Table
 from astropy.table import vstack
 from astroquery.mast import Observations
-from specutils import Spectrum1D
+from specutils import Spectrum
```

Comment:
This bumps the minimum spectutils requirement to be `specutils>=1.20` which got released 2025 May, so we may consider doing a conditional import (IMO it's fine in this specific case to require this almost latest version as specutils doesn't limit its dependencies that much, but overall this approach may not work with other packages).